### PR TITLE
docs. implemented tools section. 

### PR DESCRIPTION
**Pull request by raydecastro**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -245,6 +245,53 @@ <h3 class="module-title">Stateless CSRF Validation</h3>
...
+					<span class="sr-only">Previous</span>
+				</a>
+				<a class="right carousel-control" href="#tools-carousel" role="button" data-slide="next">
+					<span class="button-next" aria-hidden="true"></span>
+					<span class="sr-only">Next</span>
+				</a>
+			</div>
+		</div>
+	</div>
+	<!-- end electrode tools -->
```
Comment (caoyangs) [inline index.html]: format?

Comment (raydecastro) [inline index.html]: @caoyangs space vs tab issue has been fixed. my bad.

File: index.html [hunk 2]
```diff
@@ -318,5 +368,11 @@ <h3 class="module-title">Stateless CSRF Validation</h3>
   </div>
   <!-- end footer -->
 
+	<script src="https://code.jquery.com/jquery-3.1.1.slim.min.js" 
+	  integrity="sha256-/SIrNqv8h6QGKDuNoLGA4iret+kyesCkHGzVUUV0shc=" 
+		crossorigin="anonymous"></script>
+	<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" 
+		integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" 
+		crossorigin="anonymous"></script>
```
Comment (caoyangs) [inline index.html]: here:
```
    <script src="https://code.jquery.com/jquery-3.1.1.slim.min.js" 
      integrity="sha256-/SIrNqv8h6QGKDuNoLGA4iret+kyesCkHGzVUUV0shc=" 
      crossorigin="anonymous" />
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" 
      integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" 
      crossorigin="anonymous" />
```

Comment (raydecastro) [inline index.html]: @caoyangs spacing fixed.

**Pull request merged**